Sources/Hummingbird/Application.swift

```diff
@@ -122,11 +122,7 @@ extension HBApplicationProtocol {
                 configuration: .init(services: services, logger: self.logger)
             )
             try await serviceGroup.run()
-        } onGracefulShutdown: {
-            Task {
-                try await server.shutdownGracefully()
-            }
-        }
+        } onGracefulShutdown: {}
```

Comment:
I might misinterpret this, but is there any point in the graceful shutdown handler block now?

Reply:
You're right we don't need it